Lib/test/test_pathlib.py

```diff
@@ -2429,13 +2429,19 @@ def test_complex_symlinks_relative(self):
     def test_complex_symlinks_relative_dot_dot(self):
         self._check_complex_symlinks(os.path.join('dirA', '..'))
 
+    def test_class_getitem(self):
```

Comment:
This test now works for all `_BasePathTest` subclasses, previously it was only checking `PurePath`.